packages/gatsby-theme-newrelic/src/components/InteractiveForm.js

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
+import { useDebounce } from 'react-use';
+import { css } from '@emotion/react';
+import { scroller } from 'react-scroll';
+import Link from './Link';
+import Callout from './Callout';
+import CustomTextInput from './CustomTextInput';
+import InteractiveOutput from './InteractiveOutput';
+
+const InteractiveForm = () => {
+  const [appName, setAppName] = useState('My Application');
```

Comment:
Do we want to make these placeholder values in the CustomTextInputs vs initial state? I know the designs have it as a fully dark font, but that was before we updated the placeholder styles.

Reply:
hmm not sure, i kind of like that it matches the config file from the start but we can def change it!